### PR TITLE
Allow user to upload and render images

### PR DESCRIPTION
**Pull request by etcadinfinitum**
Very basic functionality that renders uploaded image onto webpage.

### CODE REVIEW

File: gui/src/templates/main.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+    <head>
+
+        <title>DOORS | DiReCtEd OpEn ObJeCt ReAliTY SiMuLaToR</title>
```
Comment (cforcomputer) [inline gui/src/templates/main.html]:
```suggestion
        <title>DOORS | Directed Object Reality Simulator</title>
```
This is a serious matter!

Comment (etcadinfinitum) [reply gui/src/templates/main.html]: Lol, fair enough. Do we want to do `Object` (`DORS`) or `Open Object` (`DOORS`)?

**Review by cforcomputer — COMMENTED** [review]
LGTM

File: gui/src/templates/main.html (hunk 1 above)
Comment (cforcomputer) [reply gui/src/templates/main.html]: Well open object is nod to the fact that it's an opening door that shows what's behind it.

**Pull request merged**